packages/react-instantsearch-hooks-dom/src/widgets/__tests__/Hits.test.tsx

```diff
@@ -0,0 +1,172 @@
+import { render, waitFor } from '@testing-library/react';
+import React from 'react';
+import { InstantSearch } from 'react-instantsearch-hooks';
+
+import { createSearchClient } from '../../../../../test/mock';
+import {
+  createMultiSearchResponse,
+  createSingleSearchResponse,
+} from '../../../../../test/mock/createAPIResponse';
+import { Hits } from '../Hits';
+
+import type { InstantSearchProps } from 'react-instantsearch-hooks';
+
+const searchClient = createSearchClient();
+
+type SearchProviderProps = {
+  children: React.ReactNode;
+} & Partial<InstantSearchProps>;
+
+function SearchProvider({ children, ...props }: SearchProviderProps) {
+  return (
+    <InstantSearch searchClient={searchClient} indexName="indexName" {...props}>
+      {children}
+    </InstantSearch>
+  );
+}
```

Comment:
We'll likely need this util for all other components, might it make sense to move it now?

Reply:
done in 469540b2